### PR TITLE
feat(web): add banneer and change placeholder

### PR DESCRIPTION
**Pull request by antonymarion**
ref: MANAGER-12708

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat #MANAGER-12708
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)

## Translations
- [x] CMT-6314

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -381,10 +381,12 @@
   "hosting_tab_databases_dumps_error_fetch": "Une erreur est survenue lors de la récupération de vos sauvegardes.",
   "hosting_tab_DATABASES_configuration_copy_title": "Sélectionnez la base de données de destination",
   "hosting_tab_DATABASES_configuration_copy_action_button": "Copier la base de données",
+  "hosting_tab_DATABASES_configuration_information_message": "Que faire si aucune base de données de destination n’apparait ?",
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:384]:
```suggestion
  "hosting_tab_DATABASES_configuration_information_message": "Que faire si aucune base de données de destination n'apparait ?",
```

Comment (darsene) [inline packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:384]:
```suggestion
  "hosting_tab_DATABASES_configuration_information_message": "Que faire si aucune base de données de destination n'apparait ?",
```

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -381,10 +381,12 @@
...
   "hosting_tab_DATABASES_configuration_copy_step1_question_part1": "Votre base de données va être copiée vers une nouvelle destination. Si aucune base de données de destination n'est disponible, vous pouvez en acheter une",
   "hosting_tab_DATABASES_configuration_copy_step1_question_part2": "et l'activer.",
   "hosting_tab_DATABASES_configuration_copy_step1_question_link": "ici",
   "hosting_tab_DATABASES_configuration_copy_step1_select_database": "Copier vers une base de données :",
+  "hosting_tab_DATABASES_configuration_copy_step1_select_hosting_placeholder": "Séléctionner un hébergement",
```
Comment (darsene) [inline packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:389]:
```suggestion
  "hosting_tab_DATABASES_configuration_copy_step1_select_hosting_placeholder": "Sélectionner un hébergement",
```

File: packages/manager/apps/web/client/app/hosting/database/copy/hosting-database-copy.controller.js [hunk 3]
```diff
@@ -22,6 +24,7 @@ angular.module('App').controller(
       this.$state = $state;
       this.$window = $window;
       this.Alerter = Alerter;
+      this.coreConfig = coreConfig;
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/hosting/database/copy/hosting-database-copy.controller.js:27]:
```suggestion
      this.ovhSubsidiary = coreConfig.getUser().ovhSubsidiary;
```

**Review by darsene — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/hosting/database/copy/hosting-database-copy.controller.js [hunk 4]
```diff
@@ -37,6 +40,11 @@ angular.module('App').controller(
     }
 
     $onInit() {
+      this.bannerGuideLink =
+        BANNER_HOSTING_GUIDE_HELP_LINKS[
+          this.coreConfig.getUser().ovhSubsidiary
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/hosting/database/copy/hosting-database-copy.controller.js:45]:
```suggestion
          this.ovhSubsidiary
```

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json (hunk 1 above)
Comment (antonymarion) [reply packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:384]: Oh dear ! I copy paste again and dit not see that thks!

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json (hunk 2 above)
Comment (antonymarion) [reply packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:389]: Oh dear ! I copy paste again and dit not see that thks!

**Review by ghyenne — DISMISSED** [review]

**Review by darsene — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**